src/modules/auth/tenant/TenantCustomDataContainer.js

```diff
@@ -1,224 +0,0 @@
-import React, { useState, useEffect } from 'react';
```

Comment:
@petrKavulok Why did you remove this container?

Reply:
`TenantCustomDataContainer.js` was the fist iteration of any custom data container and has been replaced by `CustomDataContainer.js` which is universal.

Reply:
I see